### PR TITLE
feat: update error codes, remove unused

### PR DESCRIPTION
**Pull request by Deep-Codes**

### CODE REVIEW

File: docs/javascript/v2/features/error-handling.mdx [hunk 1]
```diff
@@ -26,47 +26,45 @@ await hmsActions.unblockAudio();
...
+| **4002**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
+| **4003**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
+| **4004**       | WebRTC error                                                                                              | Some webRTC error has occured. Need more logs to debug.                                                     |
+| **4005**       | ICE Connection Failed due to network issue                                                                | Mention user to check their network connection or try again after some time.                                |
+| **5000**       | Server error for Join                                                                                     | TODO                                                                                                        |
```
Comment (Deep-Codes) [inline docs/javascript/v2/features/error-handling.mdx:50]: added `TODO` for actions

File: docs/javascript/v2/features/error-handling.mdx [hunk 2]
```diff
@@ -26,47 +26,45 @@ await hmsActions.unblockAudio();
...
+| **3003**       | Capture device is in use by some other application                                                        | Show notification to user mentioning that the capturing device is used by some other application currently. |
+| **3005**       | There is no media to return. Please select either video or audio or both.                                 | TODO                                                                                                        |
+| **3008**       | Browser has throw an autoplay exception                                                                   | Show notification to user mentioning that the browser blocked autoplay                                      |
+| **3009**       | Device unable to satisfy requested media constraints                                                      | Reduce media constraints for the role in dashboard or upgrade device hardware.                              |
+| **3010**       | No audio detected from track, indicates problem with device hardware                                      | TODO                                                                                                        |
```
Comment (raviteja83) [inline docs/javascript/v2/features/error-handling.mdx:43]: Notify user to try using a headphone for microphone. @eswarclynn this is for mic only right?

File: docs/javascript/v2/features/error-handling.mdx [hunk 3]
```diff
@@ -26,47 +26,45 @@ await hmsActions.unblockAudio();
...
+| **7002**       | Plugin Init Failed                                                                                        | Plugin initialization has failed                                                                            |
+| **7003**       | Plugin Processing Failed                                                                                  | Plugin processing failed                                                                                    |
+| **7004**       | Plugin Add Already Going on                                                                               | Plugin add is already in progress                                                                           |
+| **8001**       | Playlist: Invalid action call                                                                             | Check current playing entry, next entry, previous entry when calling seek/seekTo, playNext, playPrevious.   |
+| **8002**       | Playlist: Invalid action call                                                                             | TODO                                                                                                        |
```
Comment (raviteja83) [inline docs/javascript/v2/features/error-handling.mdx:70]: deosn't seem to be used anywhere.

Comment (Deep-Codes) [reply docs/javascript/v2/features/error-handling.mdx:70]: alright.

File: docs/javascript/v2/features/error-handling.mdx (hunk 2 above)
Comment (Deep-Codes) [reply docs/javascript/v2/features/error-handling.mdx:43]: Yep it's only for Audio/

File: docs/javascript/v2/features/error-handling.mdx (hunk 1 above)
Comment (raviteja83) [reply docs/javascript/v2/features/error-handling.mdx:50]: check the error.message for more info.

File: docs/javascript/v2/features/error-handling.mdx [hunk 4]
```diff
@@ -26,47 +26,45 @@ await hmsActions.unblockAudio();
...
+| **5000**       | Server error for Join                                                                                     | TODO                                                                                                        |
+| **5001**       | Trying to join a room which is already joined                                                             | Trying to join an already joined room.                                                                      |
+| **5002**       | Trying to join a room when preview is in progress                                                         | Call join action only when room state is "Preview" and not "Connecting".                                    |
+| **6000**       | Calling actions when not connected/joined to a room                                                       | Call the required action after joining a room.                                                              |
+| **6001**       | Unknown signalling error: {action} {error_info}                                                           | TODO                                                                                                        |
```
Comment (raviteja83) [inline docs/javascript/v2/features/error-handling.mdx:54]: I don't see this getting thrown from the sdk

File: docs/javascript/v2/features/error-handling.mdx [hunk 5]
```diff
@@ -26,47 +26,45 @@ await hmsActions.unblockAudio();
...
+| **6002**       | WebRTC Error: Error while renegotiating                                                                   | TODO                                                                                                        |
```
Comment (raviteja83) [inline docs/javascript/v2/features/error-handling.mdx:55]: same for this. not thrown from sdk.

File: docs/javascript/v2/features/error-handling.mdx [hunk 6]
```diff
@@ -26,47 +26,45 @@ await hmsActions.unblockAudio();
...
+| **6003**       | WebRTC engine is not ready yet                                                                            | TODO                                                                                                        |
```
Comment (raviteja83) [inline docs/javascript/v2/features/error-handling.mdx:56]: Wait for preview to complete before calling join. 


we should probably update the message. webRTC engine not ready seems vague

File: docs/javascript/v2/features/error-handling.mdx [hunk 7]
```diff
@@ -26,47 +26,45 @@ await hmsActions.unblockAudio();
...
+| **6004**       | Failed to parse JSON message                                                                              | Need to debug further with logs.                                                                            |
+| **6005**       | Track Metadata Missing                                                                                    | TODO                                                                                                        |
```
Comment (raviteja83) [inline docs/javascript/v2/features/error-handling.mdx:58]: this and below two are not thrown from the sdk

**Review by triptu — APPROVED** [review]

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**